oracle.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
If I run this one without any argument nothing happens which is a little surprizing.

Reply:
Now, it works without any argument